### DISCUSSION TITLE
Segmentation fault when running Python script with Phoebe?

### DISCUSSION THREAD

**Discussion by rj627** [phoebe-project/phoebe2#778] [Questions]
Hi -- I was having no issues running Phoebe until I upgraded my Python version to 3.11.4 a few days ago. I'm trying to run the [inverse examples notebook](http://phoebe-project.org/docs/2.4/examples/inverse_paper_examples) locally, and it gives a segmentation fault during the `run_compute()` line. 

I guess I'm not using the ellc compute that is suggested in the notebook (because I can't get ellc to install properly on my M1 Mac), but I was having no issues with `run_compute()` before I upgraded. Any help would be appreciated!

I've also been having issues with plotting, it seems to be complaining about a "MaskedNDArray," but that seems like an `astropy` issue and not a `phoebe` issue. Thanks again!

Python packages installed:
`python`: 3.11.4
`phoebe`: 2.4.11
`numpy`: 1.25.2
`matplotlib`: 3.7.2
`astropy`: 5.3.2

Laptop: Late 2020 M1 MacBook Pro

**Comment by kecnry** [comment]
Thanks for reporting this.  For now I would suggest using an environment with less bleeding-edge releases (especially of python/numpy/astropy) until we can get this fixed.  If you can do some testing by slowly reverting to older versions to blame a specific version of one of those three dependencies, that would be super helpful in trying to find and fix the cause.

I also created [an issue](https://github.com/phoebe-project/phoebe2/issues/779) so that we can track it and make sure it isn't forgotten.  If you happen to be able to find any more information please post it there.
